arangod/RocksDBEngine/RocksDBMetaCollection.cpp

```diff
@@ -954,81 +1076,178 @@ bool RocksDBMetaCollection::haveBufferedOperations() const {
   return false;
 }
 
-std::size_t RocksDBMetaCollection::revisionTreeDepth() const {
-  std::size_t const treeCount = _revisionTree ? _revisionTree->count() : 0;
-  std::size_t bufferInserts = 0;
-  std::size_t bufferRemovals = 0;
-  {
-    std::unique_lock<std::mutex> guard(_revisionBufferLock);
+std::unique_ptr<containers::RevisionTree> RocksDBMetaCollection::allocateEmptyRevisionTree(std::size_t depth) const {
+  // should have _revisionTreeLock held outside
+  return std::make_unique<containers::RevisionTree>(depth, _logicalCollection.minRevision().id());
```

Comment:
Indeed, the comment is now outdated. It used to be necessary before, because the function previously checked the tree's current depth. Will remove the comment.